src/components/memes/MemeList.tsx

```
@@ -0,0 +1,54 @@
+      // Pause observe when loading data
+      unobserve();
+      // Load more data
+      loadItems();
+    },
+  });
+  return (
+    <div className="flex flex-col">
+      <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-6">
+        {items.map((item, idx) => {
+          return (
+            <div ref={idx === items.length - 1 ? observe : null}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)